src/storage/mvcc/txn.rs

```diff
@@ -114,8 +126,13 @@ impl MvccTxn {
         self.modifies.push(Modify::PessimisticLock(key, lock))
     }
 
-    pub(crate) fn unlock_key(&mut self, key: Key, pessimistic: bool) -> Option<ReleasedLock> {
-        let released = ReleasedLock::new(&key, pessimistic);
+    pub(crate) fn unlock_key(
+        &mut self,
+        key: Key,
+        pessimistic: bool,
+        commit_ts: Option<TimeStamp>,
```

Comment:
Maybe we can just make it simple and consistent.

If we treat `commit_ts = 0` as rollback, can we just accept `0` instead of `None` here so as to avoid the additional `unwrap_or(0)` in `ReleasedLock::new`?